tests/dragonfly/cluster_test.py

```diff
@@ -2329,13 +2327,13 @@ async def test_migration_timeout_on_sync(df_factory: DflyInstanceFactory, df_see
     await nodes[1].client.execute_command("debug migration pause")
 
     await wait_for_error(
-        nodes[0].admin_client, nodes[1].id, "JournalStreamer write operation timeout"
+        nodes[0].admin_client, nodes[1].id, "JournalStreamer write operation timeout", 30
     )
 
     logging.debug("debug migration resume")
     await nodes[1].client.execute_command("debug migration resume")
 
-    await wait_for_status(nodes[0].admin_client, nodes[1].id, "FINISHED")
+    await wait_for_status(nodes[0].admin_client, nodes[1].id, "FINISHED", 300)
```

Comment:
this test runs around 300 seconds?
If you have 3 seconds timeout flag why do you need migration that takes this long to test this logic?

Reply:
If the test takes `300` seconds can we plz `optimize` this? Ideally `<1Min`

Reply:
it runs less than a minute, I just set a little more)